newspaper/extractors.py

```diff
@@ -213,6 +211,16 @@ def parse_date_str(date_str):
                 if datetime_obj:
                     return datetime_obj
 
+        for reg_ex in url.DATE_REGEXES:
```

Comment:
This won't work. `url` is a string input. What you meant to do was import the `DATE_REGEXES` variable from `newspaper/urls.py`.